docs/adrs/0001-aws-backup.md

```diff
@@ -0,0 +1,36 @@
+
+# Backups with AWS Backup
+
+## Status
```

Comment:
I'd vote for implicit status: everything that's non main/master is proposed, everything on main/master is implemented until deleted.